### ISSUE TITLE
Not all Packages on Networkdrive loaded

### ISSUE THREAD

**Issue by MarcelBPM** [DynamoDS/Dynamo#8246]
## Dynamo version 1.3.1

Windows 10

## What did you do? 

Installed Dynamo 1.3.1

## What did you expect to see?

All packages from the network folder loaded

## What did you see instead?

A few Packages loaded, but not all. Asked the Dynamo Forum. They suggested that it had to do with .dll blocking, but it worked perfectly normal in 1.3.0, (nothing changed on the network side), so this cannot be the issue.

**Comment by andydandy74** [comment]
https://forum.dynamobim.com/t/package-paths-on-network-drives/15399

**Comment by mjkkirschner** [comment]
@andydandy74 what do you propose here - that we improve the documentation for developers on deploying their packages?

**Comment by andydandy74** [comment]
Not my issue. Not proposing anything. Just added the link to the discussion the OP was referring to.

**Comment by radumg** [comment]
Improved documentation would be great @mjkkirschner !

At this point, we're not sure what's causing this and people's IT & network setups vary wildly, so would be good to at least understand why/what changed.

**Comment by MarcelBPM** [comment]
Another user on the forum posted this:

https://forum.dynamobim.com/t/packages-on-local-network/15888

Which might be of help

Marcel

**Comment by Lexiko** [comment]
We have the same issue. Re-pathing to anywhere on the local drive works fine. As soon as it moves to a network drive the packages disappear from the library. They still show in "manage packages" but no content.
Have tried with 1.2.1, 1.3.0 and 1.3.2.
Have tried unblocking all the files in the bin folders.
![image](https://user-images.githubusercontent.com/18419461/32081738-e713069e-bb02-11e7-8edb-79521b3bed64.png)

**Comment by mjkkirschner** [comment]
can you try having IT add your network drives as trusted sites:
https://technet.microsoft.com/en-us/library/bb496428.aspx

**Comment by mjkkirschner** [comment]
another article:
https://docs.microsoft.com/en-us/dotnet/framework/tools/caspol-exe-code-access-security-policy-tool

**Comment by mjkkirschner** [comment]
The note on that article seems to imply that something has changed between .net runtime 3 and 4 with regard to CAS - perhaps we built 1.0 against 3.x or users have now installed newer .net runtimes -  which might cause a change in behavior.

**Comment by mjkkirschner** [comment]
more info, this seems to shed some more light:
https://docs.microsoft.com/en-us/dotnet/framework/app-domains/running-intranet-applications-in-full-trust

**Comment by mjkkirschner** [comment]
last one:
https://superuser.com/questions/44503/how-do-i-tell-windows-7-to-trust-a-particular-network-location

not sure if this applies anymore with windows 10 but I would give it a shot. 

personally, I develop while storing some dlls in network drives and had to add that drive as a trusted site using IE....

**Comment by Lexiko** [comment]
Once IT added the server location to trusted sites all the packages showed up again. Thanks for the tip @mjkkirschner

**Comment by mcterry1337** [comment]
This worked for me too.

**Comment by mjkkirschner** [comment]
please reopen if using trusted sites has not resolved the issue.

**Comment by scubaman120** [comment]
Hello all,
I have completed the steps listed in @mjkkirschner "last one" link. I have done this to my computer only and the Packages with .dll files still are not there. Does this need to be done to the network server itself? 

I am the one who started the post @MarcelBPM linked above. My packages were working for about 2 weeks but as of today they no longer show up. Dynamo and Revit have not changed.

**Comment by mjkkirschner** [comment]
You must add the network share location to your trusted sites. You must also unblock the dlls.

**Comment by scubaman120** [comment]
I have added the network share location to my computers trusted sites. I just copied the file path into "Add this website to the zone:" filed and clicked Add. Is this not the correct way (It did change the file path when added)? 
![capture](https://user-images.githubusercontent.com/33437179/32471250-2804210e-c311-11e7-95c2-1791f9916d4b.PNG)

.dll files are not blocked on our network. 
![capture2](https://user-images.githubusercontent.com/33437179/32471257-2d49bc46-c311-11e7-8c1e-0ed80463fa34.PNG)

**Comment by DanBoghean** [comment]
I'm having this problem and haven't found any solutions yet. 

We've tried adding the network drives to as trusted sites and that hasn't seem to solved it. 

The other weird thing is that the .dll packages show up in Revit 2018 for me (and only me, as I have elevated privileges - they don't show up for our "normal" user), but not in Revit 2017.  I even tried putting the packages in my appdata folder and that didn't work either in Revit 2017. 

Any thoughts?
